packages/api/src/beacon/routes/beacon/rewards.ts

```diff
@@ -38,6 +40,12 @@ export type BlockRewards = {
   attesterSlashings: number;
 };
 
+/**
+ * Rewards info for sync committee participation. Every reward value is in Gwei.
+ * Note: This excludes sync aggregate reward. The reward for proposer here only reflect the sync committee participation portion
```

Comment:
Do you mean validator here?
```suggestion
 * Note: This excludes sync aggregate reward. The reward for validator here only reflects the sync committee participation portion
```

Reply:
> Do you mean validator here?

No, there is also a proposer reward for including sync committee's output to their block: https://eth2book.info/capella/part2/incentives/rewards/#proposer-rewards-for-sync-committees

The comment here is saying `SyncCommitteeRewards` only includes sync committee *participant* rewards and not *proposer* rewards for sync committee. The latter can be checked in `BlockRewards.syncAggregate`

Reply:
> If proposer index shows up in SyncCommitteeRewards, that's because proposer is also part of the sync committee, but the value here only includes participant rewards and not proposer (sync aggregate) rewards

ah that's what you meant, I didn't read it like that. It sounds a bit like the sync committee participation portion is somehow related to being a proposer